src/LibVLCSharp.MAUI/AppHostBuilderExtensions.cs

```diff
@@ -15,7 +17,18 @@ public static MauiAppBuilder UseLibVLCSharp(this MauiAppBuilder builder)
             // Register LibVLCSharp handlers
             builder.ConfigureMauiHandlers(handlers =>
             {
-                handlers.AddHandler(typeof(VideoView), typeof(VideoViewHandler));
+                //todo: add #elif for MacOS...
+
+#if ANDROID || IOS || WINDOWS
+            Debug.WriteLine($"adding VideoViewHandler for current platform: {DeviceInfo.Current.Platform}.");
```

Comment:
think this can be removed too.

Reply:
Yah, I copied this from our temp lib to get us going with libVlcsharp in Maui without libVlcsharp.Maui official package.  Had these debug prints just for sanity checking.

Removed.